apps/src/labs/header/ProjectWrappedShareDialog.tsx

```diff
@@ -0,0 +1,74 @@
+/**
+ * Wrapper around ShareDialog that plumbs in the necessary props for a project
+ * using the Project Manager and the labs system.
+ */
```

Comment:
Should we mention that this is only used by `shareProjectV2`?  Also, what exactly is the "labs system"?

Reply:
naming discussion: https://codedotorg.slack.com/archives/C044AGTKW3D/p1688154551001169